Core/Inc/main.h

```diff
@@ -49,8 +49,14 @@ extern SPI_HandleTypeDef hspi5;
 extern UART_HandleTypeDef huart1;
 extern UART_HandleTypeDef huart3;
 
-extern ETH_HandleTypeDef heth; //
-extern TIM_HandleTypeDef htim1;//
+extern ETH_HandleTypeDef heth;
+extern TIM_HandleTypeDef htim1;
+
+// for debugging: dropped messages count
+extern uint32_t GPSDroppedMessages;
```

Comment:
Why are these here? Are they used outside of their libraries?

Reply:
Thought maybe we could use them somewhere to track how much data we're losing; I should probably stick that in as #ifdef debug but let me know if it sounds useful

Reply:
I think if it is a debug utility, we would add the debugging functionality in their respective library. Tbh, I just don't want too many shared things, even if they are atomic.